### DISCUSSION TITLE
Automatically select category based on active filter

### DISCUSSION THREAD

**Discussion by vanbroup** [community/community#3138] [Discussions]
It would help is the category is automatically selected or suggested based on the current filter.

1. Say that I'm given a direct link to browse a specific category
2. As result my filter is to that category (e.g., `category:"Discussions Feedback"`)
3. I can't find what I'm looking for and want to start a new discussion 
4. Now I'm asked to select a category, but I'm not sure what to select (I did not explicitly set the filter)
5. More time is spent than needed (to select and find the correct category) and the discussion might end up in the wrong category

**Comment by evi-liu** [comment]
Thanks for the feedback! We are using this thread to prioritize and gauge customer impact via upvotes and reactions.

**Comment by PranamBhat** [comment]
Hey Van!

When you land-up on a "**Discussions**" page, you might not aware of what category it belongs to. You have only 2 categories:

1) Discussions Feedback
2) Mobile Feedback

So, nothing will be more complex here. If you are looking only specific to **Mobile Application**, then go with **2) Mobile Feedback** category.

If not, go with **1) Discussions Feedback** category. Simple :)

**Reply by vanbroup** [reply]
That doesn't work when you need to organize discussions. Categories where not created to not use them.

**Reply by PranamBhat** [reply]
What's the confusion here? We have 2 categories and 2 works for different scenarios. Both are specific to their titles.

You can always select whichever suits your discussion.

**Reply by vanbroup** [reply]
If you have just two categories that might be true, but not if you have (a few) more, say 5, 10 or 20.

And it's not about that you can but that you don't have to select it (user experience), you navigated to this category with a reason.

If the preselected category was not relevant to your question (you where looking in the wrong category), you can still see this and select something else.

**Reply by PranamBhat** [reply]
Yup! I got your point.

Imagine now, going to [https://github.com/github/feedback/discussions](https://github.com/github/feedback/discussions) page and I have something to start on (say for example : search something which is already in the discussions or create one).

1) Consider I have to search the topic/discussion before moving

Now we have "Search all discussions .. " Here we can search with "author", "category" and "is". These are the filers we can try.
We have "New", "Top", "Answered" and "Unanswered" selections. With these selections, we can easily find out our need.

2) While creating "New discussion":

We have "Discussions Feedback" and "Mobile Feedback". With these categories, one can post the discussion as per his query.

After all, once we create a "New discussion", we can go to settings and under "Category" we can always change the category once we change our mind/ if something went wrong.


Currently, in general - GitHub provided only 2 categories under "Discussions". More the category, more the confusion! That's why GitHub gave users multiple filters/selections while landing on the "Discussions" page.

**Reply by vanbroup** [reply]
I'm not talking about the GitHub usage of discussions in this feedback repository, but other repositories who can use discussions with their own set of categories.

Let's imagine you go to:
https://github.com/github/feedback/discussions/categories/mobile-feedback

Because you have some mobile feedback.

You can still filter and search as you stated but within that category.

Why would you need to select mobile feedback again when creating an new discussion?
